test/jdk/javax/swing/text/html/bug4210307.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 1999, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javax.swing.JButton;
+import javax.swing.JEditorPane;
+import javax.swing.UIManager;
+import javax.swing.text.Document;
+import javax.swing.text.Element;
+import javax.swing.text.html.FormView;
+import java.awt.Robot;
+
+/*
+ * @test
+ * @bug 4210307 4210308
+ * @key headful
+ * @summary Tests that FormView button text is internationalized
+ */
+
+public class bug4210307 {
+    private static Robot robot;
```

Comment:
Robot can be local var

Reply:
In this case robot wasn't needed hence it has been removed and the test is converted into headless.